L59-core-allow-cppstdlib.md

```diff
@@ -0,0 +1,122 @@
+Allow C++ standard library in gRPC Core Library
+----
+* Author(s): veblush
+* Approver: a11r
+* Status: Draft
+* Implemented in: n/a
+* Last updated: August 19, 2019
+* Discussion at: https://groups.google.com/forum/#!topic/grpc-io/umQyyXmNr2w
+
+## Abstract
+
+Allow C++ Standard Library to be used in the gRPC Core library.
+
+## Background
+
+Since C++ was allowed to write gRPC, more and more code has been written in C++.
+But most features from C++ standard libraries were not allowed to use because
+we didn't want to introduce new dependency against C++ standard library.
+
+The decision to use C++ has been helping us to be more productive but it's not
```

Comment:
Rephrase the two paragraph with this (has the same meaning but it more condensed)?

In the past, we allowed the use of C++ code in the gRPC Core library (link to the proposal), but we stayed conservative and avoided introducing a dependency on C++ standard library. Being able to use C++ has lead to increased productivity, but there are still significant limitations, because many useful C++ features depend on the standard C++ library.

Reply:
Thank you for the better writing :) I'll update the doc with this.